### PR TITLE
New Doc: Environment-Specific Configurations for Drupal 8

### PR DESCRIPTION
**Pull request by rachelwhitton**
Enable Twig debugging for D8 and "development mode" on pre-preoduction environments

@dustinleblanc @greg-1-anderson @peter-pantheon can you review?

### CODE REVIEW

**Comment by dustinleblanc** [comment]
👍

File: source/_docs/environment-specific-config-d8.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+<div class="alert alert-info">
+<h4>Note</h4>
+<p>Overridden configurations are not shown within Drupal's admin interface; this behavior is intentional. For details, see <a href="https://www.drupal.org/docs/8/api/configuration-api/configuration-override-system">Configuration override system</a>.</p>
+</div>
+</li></ol>
+
+4. Deploy to Test and verify desired configurations are present. If everything looks good, deploy to Live.
+
+
+Local environments can achieve the same configuration via [Drupal Console](https://drupalconsole.com/). The settings suggested on this page mirror settings applied by [`drupal site:mode dev` and `drupal site:mode prod`](https://hechoendrupal.gitbooks.io/drupal-console/content/en/commands/site-mode.html).
```
Comment (greg-1-anderson) [inline source/_docs/environment-specific-config-d8.md]: Should we document the Drupal Console way when Drupal Console is not yet supported on the platform?

File: source/_docs/environment-specific-config-d8.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+
+For more information on Pantheon's service configuration files for Drupal, refer to [Creating a services.yml File for Drupal 8](/docs/services-yml).
+
+## Override System Performance Settings Per Environment
+
+1. Use the `PANTHEON_ENVIRONMENT` constant and Drupal 8's override system within `settings.php` to enforce performance configurations based on the current Pantheon server environment:
+
+  ```php
+  if (defined('PANTHEON_ENVIRONMENT')) {
```
Comment (dustinleblanc) [inline source/_docs/environment-specific-config-d8.md]: We actually might want to unify that more across all of our docs

Comment (rachelwhitton) [inline source/_docs/environment-specific-config-d8.md]: +1 Agreed, I'll make a follow-up issue to unify usage throughout. Thanks!

**Pull request merged**